_RELEASE/Packs/cube/Scripts/Levels/seconddimension.lua

```diff
@@ -13,11 +13,11 @@ function addPattern(mKey)
 	elseif mKey ==  5 then pInverseBarrage(0)
 	elseif mKey ==  6 then pTunnel(math.random(1, 3))
 	elseif mKey ==  7 then pMirrorWallStrip(1, 0)
-	elseif mKey ==  8 then 
+	elseif mKey ==  8 then
 		if l_getSides() > 5 then
 			pWallExVortex(0, 1, 1)
 		end
-	elseif mKey ==  9 then pDMBarrageSpiral(math.random(4, 7), 0.4, 1)
+	elseif mKey ==  9 then pDMBarrageSpiral(math.random(4, 7), 0.4 * (u_getDifficultyMult() ^ 0.3), 1)
```

Comment:
This is necessary because otherwise the spiral barrage becomes impossible at `2.2`